src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageMetadataProvider.cs

```diff
@@ -247,6 +257,25 @@ private static IEnumerable<PackageVulnerabilityMetadata> MergeVulnerabilityMetad
             return vulnerabilityMetadatas.FirstOrDefault(v => v != null && v.Any());
         }
 
+        private static string MergePackagePath(IEnumerable<IPackageSearchMetadata> packages)
+        {
+            var packagePaths = packages
+                .Select(packageMetadata =>
+                {
+                    if (packageMetadata is LocalPackageSearchMetadata localPackageSearchMetadata)
```

Comment:
I guessed we had this logic somewhere, but I couldn't find it. Perhaps it could be put in a utility class, but this provider seems reasonable to me.